te3/tutorials/data-security/data-security-about.md

```diff
@@ -0,0 +1,386 @@
+    - _Can only see Sales data; they can't see [Margin %]. This is due to the OLS Object Permission `None` on 'Territory Sales'[Cost], which affects the dependent measure [Margin %]_
+2. _Janet, who has been assigned both the 'Read Users' and 'Build Users' Role:_
+    - _Cannot see any data. The combination of RLS / OLS permissions across Roles are invalid._
+
+
+_Users granted Build Permissions to the dataset are added to the Build Azure AD Security group, which is set for the 'Build Users' role. Build users can see tables not in existing reports, so `None` OLS permission is configured for the 'Employees' table. This produces a combination where RLS and OLS permissions cannot be reconciled, resulting in an error._
+
+
+> [!WARNING]
+> Scenarios combining RLS & OLS are not rare.
+> <br>Scenarios using them correctly are. 
+> <br>Ensure that if you have a requirement for RLS & OLS together, carefully consider it during model design.
+
+
+# [❌ No Roles](#tab/role)
+__No user will be able to read any data until they are added to the role__, so long as Data Security is configured in the dataset. 
+
+![No Access or No Role](~/images/data-security/data-security-no-role.png)
+
+> [!NOTE]
+> Don't forget to give users access to the dataset AND add them to the security role.
+
+
+# [❌ No Access](#tab/access)
+__If a user is added to a security role, this will not automatically grant them read access to the dataset.__ They will still not be able to access any dataset or reports. 
+
+![No Access or No Role](~/images/data-security/data-security-no-access.png)
+
+> [!NOTE]
+> Don't forget to give users access to the dataset AND add them to the security role.
+
+> [!WARNING]
+> __It is a best practice to avoid distribution via Workspace Roles, where feasible.__
+> <br> If necessary, ensure that the __Principle of Least Permissions__ is applied : users should have the minimum access necessary to do what they need.
+
+
+# [❌ Access Via Workspace Roles](#tab/workspace)
+__If a user is given access to a dataset via the Admin, Member or Contributor roles, they will be able to view all data, irrespective of the data security configuration and their assigned roles.__ This is a common mistake in scaling or self-service Power BI ecosystems, resulting in data leaks and non-compliance. 
+
+![Problems with ](~/images/data-security/data-security-workspace-roles.png)
+
+> [!WARNING]
+> __It is a best practice to avoid distribution via Workspace Roles, where feasible.__
+> <br> If necessary, ensure that the __Principle of Least Permissions__ is applied : users should have the minimum access necessary to do what they need.
+
+---
+
+---
+
+### Hard Limitations
+Some reporting or data modeling features will not work with RLS or OLS configuration. Examples are:
+
+1. [RLS Limitations](https://learn.microsoft.com/en-us/power-bi/enterprise/service-admin-rls#considerations-and-limitations)
+    - Power BI - Analysis Services Live Connections
```

Comment:
You cannot configure RLS in a power bi artifact that has live connection with Analysis Services, since the RLS is handled in the AS model. That is what is meant in the Power BI documentation. I think I will just remove this limitation because it is confusing; its in the PBI docs so that the user understands they shouldnt set up anything additional on the PBI side.